### ISSUE TITLE
[nodejs] Fetch multiple calendars from multiple accounts.

### ISSUE THREAD

**Issue by SaudTauqeer** [googleapis/google-api-nodejs-client#1990]
I need some help/guidance regarding an issue I am facing for a couple of days and have not been able to reach a conclusive solution.
#Problem
I am trying to let a user sign up with a google account (already have that setup) after the user has logged in it's set as the main account in my DB.
I want to fetch that user's google calendar events (and be able to keep fetching them i.e polling for new calendar events) and let the authenticated user add more google accounts without logging out of his "main" google account and then fetch google calendar events of those accounts that the user keeps adding.
#My research so far.
- I have looked into a service account but I'm kind of confused about how to pull this off with that. I have read into most places that the user needs to set the calendar to the public and then share my app's service email and then I can read/write those events. 
- if I store access token, in the docs it says the max time limit is 1 hour and that means I would have to ask for access again.
#TLDR: Looking for a way to add persistent access to multiple google calendar events without logging out the other account (read-only is fine).
